Data/ByteString.hs

```diff
@@ -118,6 +118,7 @@ module Data.ByteString (
         takeWhileEnd,           -- :: (Word8 -> Bool) -> ByteString -> ByteString
         dropWhile,              -- :: (Word8 -> Bool) -> ByteString -> ByteString
         dropWhileEnd,           -- :: (Word8 -> Bool) -> ByteString -> ByteString
+        dropEnd,                -- :: Int -> ByteString -> ByteString
```

Comment:
Could you please put it right after `drop`? Cf. the order of exports in `Text`: http://hackage.haskell.org/package/text-1.2.2.1/docs/Data-Text.html#v:dropEnd